src/main/java/no/ssb/subsetsservice/SubsetsController.java

```diff
@@ -66,7 +68,25 @@ public SubsetsController(){
 
     @GetMapping("/v1/subsets/{id}/versions")
     public ResponseEntity<String> getVersions(@PathVariable("id") String id) {
-        return getFrom(LDS_SUBSET_API, "/"+id+"?timeline");
+        ResponseEntity<String> timelineResponseEntity = getFrom(LDS_SUBSET_API, "/"+id+"?timeline");
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            JsonNode responseBodyJSON = mapper.readTree(timelineResponseEntity.getBody());
+            if (responseBodyJSON != null){
+                if (responseBodyJSON.isArray()) {
+                    ArrayNode responseBodyArrayNode = (ArrayNode) responseBodyJSON;
+                    for (int i = 0; i < responseBodyArrayNode.size(); i++) {
```

Comment:
Tip: try java 8 forEach. 

- `responseBodeArrayNode.strream().foreach(arrayEntry -> arrayEntry.get("document").set("_links"))`
- `for(ObjectNode arrayEntry : responseBodeArrayNode) {
	arrayEntry.get("document").set("_links");
}`